Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:slim-bullseye
```

Comment:
FROM python:3.10-slim-bullseye

pytorchvision cannot be installed if the version of python is > 3.10